css/main.css

```diff
@@ -325,3 +325,82 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+footer {
+  height: 500px;
+  margin-top: 3%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+}
+
+.form-container {
+  /* display: flex;
+  justify-content: space-between;
+  width: 90%; */
+  margin-left: 5%;
+}
+
+.form-container p {
+  /* width: 35%; */
```

Comment:
- [ ] It would be better if the CSS file didn't have commented code in order to make your codebase cleaner and readable :white_check_mark: :white_check_mark: :white_check_mark:

Reply:
Good job at integrating this change ✔😄